apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,125 @@
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  Text,
+  IconButton,
+  NotificationMarker,
+  NotificationMarkerProps,
+} from '@itwin/itwinui-react';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  component: NotificationMarker,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    type: {
+      control: 'radio',
+      options: ['primary', 'positive', 'warning', 'negative'],
+    },
+  },
+  title: 'Core/NotificationMarker',
+} as Meta<NotificationMarkerProps>;
+
+export const Basic: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <IconButton styleType='borderless'>
+      <NotificationMarker {...args}>{args.children}</NotificationMarker>
+    </IconButton>
+  );
+};
+
+Basic.args = {
+  type: 'primary',
+  urgent: false,
+  children: <SvgNotification />,
+};
+
+Basic.argTypes = {
+  children: {
+    control: false,
+  },
+};
+
+export const CustomChild: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        gap: '20px',
+      }}
+    >
+      <NotificationMarker type='positive' {...args}>
+        Live
+      </NotificationMarker>
+      <NotificationMarker type='negative' {...args}>
+        Rec
+      </NotificationMarker>
+      <NotificationMarker type='primary' {...args}>
+        1 new message
+      </NotificationMarker>
+    </div>
+  );
+};
+
+CustomChild.args = {
+  urgent: true,
+};
+
+CustomChild.argTypes = {
+  type: {
+    control: false,
+  },
+};
+
+export const ProgrammaticDisplay: Story<
+  NotificationMarkerProps & { demoNotificationCount: number }
+> = (args) => {
+  const active = args.demoNotificationCount > 0;
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        gap: '20px',
+      }}
+    >
+      <div>
+        <Text variant='heading'>Try changing demoNotificationCount</Text>
+        <Text variant='small'>active = demoNotificationCount &gt; 0</Text>
+      </div>
+      <IconButton styleType='borderless'>
+        <NotificationMarker active={active} {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+      <NotificationMarker active={active} {...args}>
+        <Text>
+          {args.demoNotificationCount > 0
+            ? 'New notification(s)'
+            : `No new notifications`}
+        </Text>
+      </NotificationMarker>
+    </div>
+  );
+};
+
+ProgrammaticDisplay.args = {
+  demoNotificationCount: 0,
+  urgent: true,
+  type: 'primary',
+};
+
+ProgrammaticDisplay.argTypes = {
+  active: {
+    control: false,
+  },
+};
```

Comment:
I added uses of `<NotificationMarker>` in `Header`'s Full story ([commit](https://github.com/iTwin/iTwinUI-react/pull/829/commits/3b029f0c1df338e65334384f13d79c8c7168d3b7)) and `SideNavigations`'s Basic story ([commit](https://github.com/iTwin/iTwinUI-react/pull/829/commits/bee281d467f8697e832763e203652837999daa8e)).

I did not add any props to modify the `NotificationMarker`s since it might be confusing whether `status` and `pulsing` are parameters of `Header`/`SideNavigation` or `NotificationMarker`.

To confirm, is the way I have added it alright?

Reply:
Don't forget to update the story test images for Header and SideNavigation.

Reply:
**[DONE]** Thanks, approved Header and SideNavigation images